### DISCUSSION TITLE
Rework a bit combat balance

### DISCUSSION THREAD

**Discussion by juju59710** [endless-sky/endless-sky#9817] [Ideas]
Hello everyone,

I may have an interesting idea for a rework on some aspects of space combats, mainly fighters and weapons. (I know this has already been talked about, but I am here more for the last paragraph)

First, the fighters : right now, I thnik we can all agree they are quite useless. But I may have an idea to change that : what if we made the fighters a bit like missiles and torpedoes, in their own category ? What I mean by that is that in order to engage fighters, instead of regular weapons you would need "anti-fighters" guns to attack them. It could use a different category of weapons (using the "attributes" that are already in-game) so that it doesn't restrict the offensive capabilities of a ship. It would also give a counter to massive warships : they would have fewer anti-fighter weapons, meaning that they could deal less easily with them. 
In the same direction, we could have "classes" of fighters : space superiority and bombers for exemple. 

In terms of weapons, we could add heavy weapons that require specific slots (still using attributes) that are more effective against slow moving targets. Not really much to add, i think you see the idea.

What I am asking with this post isn't really to have it implemented, but more how could I access some files that could make this real. I already went through the folders,  and what I am searching for isn't in the data files. I suspect they are inside the DLLs, but I can't figure out how to access them.

If anyone could help me with that, I would be grateful.

**Comment by Zitchas** [comment]
Part of that will take changes to the code/mechanics, which you can find in the source folder in the repository here. That will be the hard part. The rest is in the data, which will be in each factions folder, typically in a file called "outfits.txt" or "weapons.txt" or similar.

**Reply by juju59710** [reply]
Thanks for the reply, I'll see what I can do to (hopefully) make something that works.

**Reply by juju59710** [reply]
I am wondering, now that I can edit things, is there a way I can lauch the game to see if things work correctly in-game ? I am quite new to this, and I don't know how to put my modifications in action. Thanks in advance !

**Reply by Zitchas** [reply]
Data changes will be immediately reflected when you relaunch the game.

Code changes, however, will require you to compile a new executable. Instructions on doing that are in the wiki.
